src/CMakeLists.txt

```diff
@@ -1,62 +1,71 @@
-include(MacroQbtCompilerSettings)
-qbt_set_compiler_options()
-
-include(QbtTargetSources)
-
-find_package(Boost ${requiredBoostVersion} REQUIRED)
-find_package(LibtorrentRasterbar ${requiredLibtorrentVersion} REQUIRED)
-find_package(OpenSSL ${requiredOpensslVersion} REQUIRED)
-
-if (Boost_VERSION_STRING VERSION_LESS 1.60.0)
-    add_definitions(-DBOOST_NO_CXX11_RVALUE_REFERENCES)
+set(LibtorrentRasterbar_DIR "" CACHE PATH "Path to LibtorrentRasterbar CMake configuration files")
+find_package(LibtorrentRasterbar QUIET ${requiredLibtorrentVersion} COMPONENTS torrent-rasterbar)
+if (UNIX AND (NOT APPLE) AND (NOT CYGWIN))
+    if (NOT LibtorrentRasterbar_FOUND)
+        include(FindPkgConfig)
+        pkg_check_modules(LIBTORRENT_RASTERBAR IMPORTED_TARGET GLOBAL "libtorrent-rasterbar>=${requiredLibtorrentVersion}")
+        if (NOT LIBTORRENT_RASTERBAR_FOUND)
+            message(FATAL_ERROR "Package LibtorrentRasterbar >= ${requiredLibtorrentVersion} not found"
+            " with CMake or pkg-config.\n- Set LibtorrentRasterbar_DIR to a directory containing"
```

Comment:
You should add one more indentation level to this and folloving lines (until the end of "message" parameter list).

Reply:
@glassez All such instances fixed.